opentelemetry-sdk/README.md

```diff
@@ -34,3 +34,8 @@ From the root directory, run the following command:
 ```sh
 cargo bench
 ```
+
+## Supported Rust Versions
```

Comment:
we need to put it in main readme, and everywhere else can just point to it

Reply:
Yeah I thought so, but then kept it consistent with other crates from this repo. 

https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-rust+%22Supported+Rust+Versions%22&type=code